property_tests/tests/float32/test/test.ts

```diff
@@ -66,11 +66,7 @@ function generateBody(
     );
     const paramsCorrectlyOrdered = paramNames
         .map((paramName, index) => {
-            const areFloat32sEqual = areFloatsEqual(
-                paramName,
-                paramLiterals[index]
-            );
-            return `if (!(${areFloat32sEqual})) throw new Error('${paramName} is incorrectly ordered')`;
+            return `if (!deepEqual(${paramName}, ${paramLiterals[index]})) throw new Error('${paramName} is incorrectly ordered')`;
```

Comment:
Abstract this away since it should be the same check for all tests, write this down.

Reply:
What if instead of writing it down, I just did it real quick? @lastmjs

Reply:
I think I'd get a kick out of that